docs/sources/mimir/configure/about-versioning.md

```diff
@@ -123,6 +123,7 @@ The following features are currently experimental:
   - Lower TTL for cache entries overlapping the out-of-order samples ingestion window (re-using `-ingester.out-of-order-allowance` from ingesters)
   - Use of Redis cache backend (`-query-frontend.results-cache.backend=redis`)
   - Query blocking on a per-tenant basis (configured with the limit `blocked_queries`)
+  - Backoff and retry requests received while the query-frontend is starting up (`-query-frontend.not-running-backoff`)
```

Comment:
(nit) this reads as an enablement for backing-off rather than setting the backoff period.

Reply:
Good point, I've renamed the flag to `-query-frontend.not-running-backoff-duration` to make it clearer.